internal/api/handlers/history_activity_test.go

```
@@ -0,0 +1,281 @@
+		wantBucket string
+	}{
+		{name: "defaults", window: "", bucket: "", wantErr: false, wantWin: "24h", wantBucket: "hour"},
+		{name: "24h_hour", window: "24h", bucket: "hour", wantErr: false, wantWin: "24h", wantBucket: "hour"},
+		{name: "7d_day", window: "7d", bucket: "day", wantErr: false, wantWin: "7d", wantBucket: "day"},
+		{name: "invalid_window", window: "30d", bucket: "day", wantErr: true},
+		{name: "invalid_24h_bucket", window: "24h", bucket: "day", wantErr: true},
+		{name: "invalid_7d_bucket", window: "7d", bucket: "hour", wantErr: true},
+	}
+
+	for _, tc := range tests {
+		tc := tc
```

golangci-lint (2.11.4)

[error] The copy of the 'for' variable "tc" can be deleted (Go 1.22+)

(copyloopvar)